websites/K/Kick/presence.ts

```diff
@@ -0,0 +1,65 @@
+const presence = new Presence({ clientId: "1112901248421732462" });
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/kVabJjN.png",
+			startTimestamp: Date.now(),
```

Comment:
This likely does not have the effect you are looking for. Try something like this:
```ts
const browsingTimestamp = Math.floor(Date.now() / 1000);

presence.on("UpdateData", () => {
  // ...
    startTimestamp: browsingTimestamp,
  }
  // ...
});
```